site/templates/nav.jet

```diff
@@ -2,73 +2,67 @@
 {{import "./nav/user_logged_in.jet"}}
 
 {{block nav(showSubNav=true)}}
-<div class="navigation" data-toggle="affix">
-  <nav class="navbar navbar-expand-md navbar-dark">
-      <a class="navbar-brand" href="{{ routeToPath("/") }}">{{i18n("nav_homepage")}}</a>
-      
+<div class="navigation search-hidden" data-toggle="affix" aria-hidden="true">
+  <nav>
+
+    <div class="navbar">
+      <div class="navbar-brand-container">
+        <a class="navbar-brand" href="{{ routeToPath("/") }}">{{i18n("nav_homepage")}}</a>
+      </div>
+
       {{if config("search_disabled") != "true"}}
-        <form class="form-search navbar-nav-search" action="{{ routeToPath("/search.html") }}">
-            <input class="form-control form-control-search" type="search" placeholder="{{i18n("search_control_placeholder")}}" name="q" aria-label="{{i18n("search_control_placeholder")}}">
-            <button class="sr-only" type="submit">{{i18n("search_control_submit")}}</button>
+        <form class="form-search navbar-nav-search search-desktop"  action="{{ routeToPath("/search.html") }}">
+          <button type="button" class="search-open" onclick="focusSearch()">
+            <svg class="search-icon" width="24" height="24" viewBox="0 0 24 24"><path d="M9.5,3A6.5,6.5 0 0,1 16,9.5C16,11.11 15.41,12.59 14.44,13.73L14.71,14H15.5L20.5,19L19,20.5L14,15.5V14.71L13.73,14.44C12.59,15.41 11.11,16 9.5,16A6.5,6.5 0 0,1 3,9.5A6.5,6.5 0 0,1 9.5,3M9.5,5C7,5 5,7 5,9.5C5,12 7,14 9.5,14C12,14 14,12 14,9.5C14,7 12,5 9.5,5Z" /></svg>
```

Comment:
We use Font Awesome 4 for icons... no need for svg chaos!

Reply:
Design specifically wanted icons from here: (https://materialdesignicons.com) and to be fair, the Font Awesome icons look pretty terrible. I do agree that inserting them as SVG's is not the greatest though. Do you know if we have the pro version of font awesome? Because then we could upload any icons we want. Alternativley, I could add them as blocks in a .jet file and use `yield` to call them. That would tidy it up.

Reply:
Hmm okay... then I think that we should have a separate task to change all icons across the site (might need some Relish work too) instead of changing a few for the nav task.  And I'm pretty sure that MDI icons just need classes like Font Awesome... no SVG lol.

Reply:
Replaced with Font Awesome